### PR TITLE
Removing old PHP Version checks

### PR DESCRIPTION
**Pull request by scottcwilson**
Part 2 of #2586.  

Note that some of the removed checks were actually wrong; they used OR where they should have used AND.

### CODE REVIEW

File: admin/includes/functions/general.php [hunk 1]
```diff
@@ -909,7 +909,7 @@ function zen_set_product_status($products_id, $status) {
 // Sets timeout for the current script.
 // Cant be used in safe mode.
   function zen_set_time_limit($limit) {
-    if (version_compare(PHP_VERSION, 5.4, '>=') || !get_cfg_var('safe_mode')) {
+    if (!get_cfg_var('safe_mode')) {
```
Comment (drbyte) [inline admin/includes/functions/general.php]: do we even need the `safe_mode` check anymore?

Comment (scottcwilson) [reply admin/includes/functions/general.php]: I do see it once in a while.  (Of course I tell the client to switch, but I do see it.)

Comment (drbyte) [reply admin/includes/functions/general.php]: But, in the sqlpatch update you removed it. 
Let's be consistent and remove it in both places. It was deprecated in PHP 5.3 and removed in PHP 5.4

Comment (scottcwilson) [reply admin/includes/functions/general.php]: LOL you're right.  So much for keeping track of time.  OK removed.

**Pull request merged**